### PR TITLE
Use config file for chains, add set-config step for dynamically generated YAML

### PR DESCRIPTION
**Pull request by 0xCaliburSigma**

### CODE REVIEW

File: src/config/chains.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  arbitrumOne,
+  arbitrumSepolia
+)
+
+// Edit this for the given deployment
+const TARGET_CHAIN: ChainInfo = supportedChains.arbitrumOne
+
+export {
+  TARGET_CHAIN,
+  ARBITRUM_ONE
```
Comment (0xMyrddin) [inline src/config/chains.ts:88]: Just to confirm, should we leave this one as is when deploying to another chain?

File: src/mappings/pricing.ts [hunk 2]
```diff
@@ -19,25 +24,6 @@ export function getEthPriceInUSD(): BigDecimal {
...
-  '0x6cda1d3d092811b2d48f7476adb59a6239ca9b95', // stafi-rETH
-  '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9', // USDT
-  '0xda10009cbd5d07dd0cecc66161fc93d7c9000da1', // DAI
-  '0xeb1a8f8ea373536600082ba9ae2db97327513f7d', // apOHM
-]
-
-let STABLE = '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8'
-
 // minimum liquidity required to count towards tracked volume for pairs with small # of Lps
 let MINIMUM_USD_THRESHOLD_NEW_PAIRS = BigDecimal.fromString('500')
```
Comment (0xMyrddin) [inline src/mappings/pricing.ts:28]: Thresholds should be dynamically set too

Comment (0xMyrddin) [reply src/mappings/pricing.ts:28]: `MINIMUM_USD_THRESHOLD_NEW_PAIRS`, `MINIMUM_LIQUIDITY_THRESHOLD_ETH` and `MINIMUM_LIQUIDITY_ETH` won't be the same for each chain

File: src/config/chains.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  arbitrumOne: ChainInfo;
+  arbitrumSepolia: ChainInfo;
+
+  constructor(arbitrumOne: ChainInfo, arbitrumSepolia: ChainInfo) {
+    this.arbitrumOne = arbitrumOne;
+    this.arbitrumSepolia = arbitrumSepolia;
+  }
+}
+
+const arbitrumOne = new ChainInfo(
```
Comment (0xMyrddin) [inline src/config/chains.ts:43]: Aside from the Factory, all those addresses (native, WL, etc...) HAVE to be in lowercase. Forcing them into lowercase when applying the conf would avoid potential issues

File: src/config/chains.ts (hunk 1 above)
Comment (0xCaliburSigma) [reply src/config/chains.ts:88]: in order for this to work we need to add the chain to SupportedChains, create a new ChainInfo for it and set to TARGET_CHAIN

**Review by 0xMyrddin — APPROVED** [review]

**Pull request merged**